WalletWasabi.Fluent/Extensions/DynamicDataMixin.cs

```diff
@@ -0,0 +1,66 @@
+using System.Reactive.Linq;
+using DynamicData;
+using DynamicData.Kernel;
+
+namespace WalletWasabi.Fluent.Extensions;
+
+public static class Mixin
```

Comment:
This is an upcoming feature in DynamicData. This class is needed because it's not yeat added to the official package.